index.js

```diff
@@ -33,14 +34,8 @@ module.exports = function(options) {
     requireFn = function (name) {
       // This searches up from the specified package.json file, making sure
       // the config option behaves as expected. See issue #56.
-      var searchFor = path.join('node_modules', name);
-
-      var src = findup(searchFor, {cwd: path.dirname(config)}) || name;
-      if (src !== null) {
-        return require(src);
-      } else {
-        throw new Error('Cannot find `' + name + '` in your node_modules!');
-      }
```

Comment:
There's already a test for it [here](https://github.com/jackfranklin/gulp-load-plugins/blob/master/test/index.js#L202-L208). It's not _exactly_ the same error, but it's close enough that it doesn't break the test—and this PR introduces a breaking change anyway, so I can't see the point in wrapping it in a try catch block just to change the wording of the error slightly.